test/geochem/geochem_alteration.py

```diff
@@ -87,5 +87,33 @@ def test_WIP(self):
         df.loc[:, "WIP"] = WIP(df)
 
 
+class TestAI(unittest.TestCase):
+    """Tests the Alteration index."""
+
+    def setUp(self):
+        self.cols = ["SiO2", "CaO", "MgO", "FeO", "TiO2", "Na2O", "K2O", "Al2O3"]
+        self.df = pd.DataFrame(
+            {k: v for k, v in zip(self.cols, np.random.rand(len(self.cols), 10))}
+        )
+
+    def test_AI(self):
+        df = self.df
+        df.loc[:, "AI"] = AI(df)
```

Comment:
A test will fail here due to some changes in recent `pandas` versions. You can update it with:
```python
df["AI"] = AI(df)
```